app/tasks.py

```diff
@@ -379,3 +380,65 @@ def make_query(self):
     def update_counter(self, counter, person):
         person.update_index(['old', 'new'])
         person.put()
+
+
+class NotifyBadReviewStatus(utils.BaseHandler):
+    """This task sends email notification when the number of unreviewed notes
+    exceeds threshold.
+    """
+    ACTION = 'tasks/notify_bad_review_status'
+
+    def task_name(self):
+        return 'notify-bad-review-status'
+
+    def get(self):
+        if self.repo:
+            try:
+                count_of_unreviewed_notes = \
+                    model.Note.unreviewed_record_count(self.repo)
+            except runtime.DeadlineExceededError:
+                logging.info("DeadlineExceededError occurs")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            except datastore_errors.Timeout:
+                logging.info("Timeout occurs in datastore")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            finally:
+                self._notify(count_of_unreviewed_notes)
```

Comment:
If the exceptions above are raised, you don't want to cal self._notify right? Then you should  put this line inside the try-clause above instead of this finally-clause. finally-clause will be always executed regardless of the exceptions.